File: promedio.py
archivo=open("Promedio.txt", "a")
x=int(input("Ingrese la primera nota: "))
y=int(input("Ingrese la segunda nota: "))
z=int(input("Ingrese la tercera nota: "))

prom=(x+y+z)/3

if (prom> 60):
    print("APROBADO ", prom)
    archivo.write('APROBADO ' + 'prom=% s' %prom + '\n')
else:
    print("REPROBADO: ",prom)
    archivo.write("REPROBADO " + 'prom=% s' %prom + '\n')

archivo.close ()

File: promedio.m
archivo=fopen('Promedio.txt','a');

valor=input('ingrese la primera nota: ','s');
n=str2num(valor);
valor=input("ingrese la segunda nota: ", 's');
m=str2num(valor);

valor=input("ingrese la tercera nota: ",'s');
p=str2num(valor);

resultado=(n+m+p)/3;
disp(['el promedio es ' ,num2str(resultado)])
fprintf(archivo,'el promedio es: %d',resultado);
printf('Registro guardado \n');

if (resultado > 60)
  disp('Aprobado')
  fprintf(archivo,' Aprobado \n');
else
  disp('Reprobado')
  fprintf(archivo,' Reprobado \n');
endif

fclose(archivo);